### DISCUSSION TITLE
lc/2339/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#1821] [Announcements]
# lc/2339/

多种编程语言实现 LeetCode、《剑指 Offer（第 2 版）》、《程序员面试金典（第 6 版）》题解

https://leetcode.doocs.org/lc/2339/

**Comment by thinkasany** [comment]
似乎可以不用 CROSS，也能通过 @yanglbme

**Reply by yanglbme** [reply]
那你帮忙简化一下写法
